dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml

```diff
@@ -651,6 +801,11 @@
       <artifactId>smallrye-graphql</artifactId>
       <version>1.0.26</version>
     </dependency>
+    <dependency>
```

Comment:
Do we need this `zipkin` dependency if we use `io.opentelemetry:opentelemetry-exporter-zipkin`?